core/modules.rs

```diff
@@ -459,29 +459,6 @@ impl ModuleLoader for ExtModuleLoader {
   }
 }
 
-impl Drop for ExtModuleLoader {
-  fn drop(&mut self) {
-    let sources = self.sources.get_mut();
-    let used_specifiers = self.used_specifiers.get_mut();
-    let unused_modules: Vec<_> = sources
-      .iter()
-      .filter(|(k, _)| !used_specifiers.contains(k.as_str()))
-      .collect();
-
-    if !unused_modules.is_empty() {
-      let mut msg =
-        "Following modules were passed to ExtModuleLoader but never used:\n"
-          .to_string();
-      for m in unused_modules {
-        msg.push_str("  - ");
-        msg.push_str(m.0);
-        msg.push('\n');
-      }
-      panic!("{}", msg);
-    }
-  }
-}
```

Comment:
I don't understand why this was removed - it was very handy to assert we don't have files that were listed for inclusion in the snapshot but were never used. Could you explain it a bit @nayeemrmn?

Reply:
The method didn't work anymore because there were now some redundant inclusions in `cli/build.rs`, and it seemed like we have a similar check at https://github.com/denoland/deno/blob/82dd90f98d6817caadcb9b6bd81aba80b2ee848d/core/runtime.rs#L895

Reply:
Okay, I guess that makes sense.